internal/pkg/rbldap/ldap.go

```diff
@@ -0,0 +1,43 @@
+package rbldap
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/redbrick/rbldap/pkg/rbuser"
+	"github.com/urfave/cli"
+)
+
+func newRbLdap(ctx *cli.Context) (*rbuser.RbLdap, error) {
```

Comment:
I would add a TODO to not use `cli.Context`, in the future, seeing the maintenance track records of redbrick projects a TODO will help to show what is wrong, in case aliens abduct you and some poor admin takes care of it in the future.

This is also a prefect example where passing in `url.URL` would make much more sense ;)